DS13/code/game/movietitles.dm

```diff
@@ -0,0 +1,83 @@
+//Makes end credits happen after round end!
+//Inspired by baystation, optimized for /tg/
+
+/datum/asset/simple/credits
+	assets = list("credits.gif"	= 'DS13/icons/stars2.gif')
+
+/proc/names2credits()
+	var/credits = "Starring the crew of [station_name()]<br>"
+	for(var/mob/living/M in GLOB.player_list)
+		if(!isliving(M) || !M.mind || !M.client)
+			continue //Ghosts don't get credit
+		credits += "[M.client.ckey] as [M.mind.assigned_role] [M.name]<br>"
+	credits += "Directed by: Kmc2000 <br> Executive producer: Francinum <br> Technical supervisor: Bass-ic <br>"
+	return credits
+
+/proc/roll_credits()
```

Comment:
Please change the name of this proc or the other one as its confusing to which does what